### ISSUE TITLE
AjaxTab.getLazyPanel() maybe leads into SerializationProblems determined by Wickets OrphanComponentChecker

### ISSUE THREAD

**Issue by Patrick1701** [sebfz1/wicket-jquery-ui#67]
Hi Sebastien,
we are using Wickets OrphanComponentChecker in a own DevelopmentSerializerCheck class in wicket development mode to get notified about SerializationProblems in our Application.

I got the following trace today while coding...
Possibly interesting for you and the AjaxTab?

kind regards
Patrick

Unexpected RuntimeException

Last cause: A component without a parent is detected.
A problem occurred while checking object with type: org.apache.wicket.markup.html.basic.Label
Field hierarchy is:
9 [class=de.xyz.MyPage, path=9]
private java.lang.Object org.apache.wicket.MarkupContainer.children [class=[Ljava.lang.Object;]
private com.googlecode.wicket.jquery.core.ajax.JQueryAjaxBehavior com.googlecode.wicket.jquery.ui.widget.accordion.AccordionBehavior.activateEventBehavior[2][3][26] [class=de.xyz.MyPanel, path=9:catalogs]
private java.lang.Object org.apache.wicket.MarkupContainer.children [class=com.googlecode.wicket.jquery.ui.widget.tabs.TabbedPanel, path=9:catalogs:catalogs]
java.lang.Object org.apache.wicket.Component.data [class=[Ljava.lang.Object;]
java.lang.Object org.apache.wicket.Component.data[0] [class=org.apache.wicket.model.util.WildcardListModel]
private java.lang.Object org.apache.wicket.model.util.GenericBaseModel.object [class=java.util.ArrayList]
private java.lang.Object org.apache.wicket.model.util.GenericBaseModel.object[write:1] [class=com.googlecode.wicket.jquery.ui.widget.tabs.AjaxTab]
private com.googlecode.wicket.jquery.core.panel.LoadingPanel com.googlecode.wicket.jquery.ui.widget.tabs.AjaxTab.panel [class=com.googlecode.wicket.jquery.core.panel.LoadingPanel, path=9:catalogs:catalogs:panels:tab-panels124-1]
private org.apache.wicket.markup.html.basic.Label com.googlecode.wicket.jquery.core.panel.LoadingPanel.label [class=org.apache.wicket.markup.html.basic.Label, path=lazy] <----- field that is causing the problem
WicketMessage: Exception in rendering component: [Component id = data]

Stacktrace

Root cause:

org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream$ObjectCheckException: A component without a parent is detected.
A problem occurred while checking object with type: org.apache.wicket.markup.html.basic.Label
Field hierarchy is:
9 [class=de.xyz.MyPage, path=9]
private java.lang.Object org.apache.wicket.MarkupContainer.children [class=[Ljava.lang.Object;]
private com.googlecode.wicket.jquery.core.ajax.JQueryAjaxBehavior com.googlecode.wicket.jquery.ui.widget.accordion.AccordionBehavior.activateEventBehavior[2][3][26] [class=de.xyz.MyPanel, path=9:catalogs]
private java.lang.Object org.apache.wicket.MarkupContainer.children [class=com.googlecode.wicket.jquery.ui.widget.tabs.TabbedPanel, path=9:catalogs:catalogs]
java.lang.Object org.apache.wicket.Component.data [class=[Ljava.lang.Object;]
java.lang.Object org.apache.wicket.Component.data[0] [class=org.apache.wicket.model.util.WildcardListModel]
private java.lang.Object org.apache.wicket.model.util.GenericBaseModel.object [class=java.util.ArrayList]
private java.lang.Object org.apache.wicket.model.util.GenericBaseModel.object[write:1] [class=com.googlecode.wicket.jquery.ui.widget.tabs.AjaxTab]
private com.googlecode.wicket.jquery.core.panel.LoadingPanel com.googlecode.wicket.jquery.ui.widget.tabs.AjaxTab.panel [class=com.googlecode.wicket.jquery.core.panel.LoadingPanel, path=9:catalogs:catalogs:panels:tab-panels124-1]
private org.apache.wicket.markup.html.basic.Label com.googlecode.wicket.jquery.core.panel.LoadingPanel.label [class=org.apache.wicket.markup.html.basic.Label, path=lazy] <----- field that is causing the problem
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.internalCheck(CheckingObjectOutputStream.java:387)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.check(CheckingObjectOutputStream.java:361)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.checkFields(CheckingObjectOutputStream.java:645)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.internalCheck(CheckingObjectOutputStream.java:569)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.check(CheckingObjectOutputStream.java:361)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.checkFields(CheckingObjectOutputStream.java:645)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.internalCheck(CheckingObjectOutputStream.java:569)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.check(CheckingObjectOutputStream.java:361)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream.access$500(CheckingObjectOutputStream.java:61)
at org.apache.wicket.core.util.objects.checker.CheckingObjectOutputStream$1InterceptingObjectOutputStream.replaceObject(CheckingObjectOutputStream.java:526)
at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1141)
at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:347)
.
.
.
